.gitignore

```diff
@@ -101,6 +101,7 @@ target/
 .python-version
 
 .env
+.envrc
```

Comment:
not sure what this is excluding?

Unless it's something very common, i'd rather not exclude stuff that's only appearing on your system.